update-compose.yaml

```diff
@@ -0,0 +1,9 @@
+policies:
+  # The following policy is designed to work from GitHub action workflows.
+  # This means that before running any Updatecli command, we need the two following environment variables set:
+  # 
+  #   GITHUB_TOKEN: Set to a personal access token
+  #   GITHUB_ACTOR: Set to the username associated with the GITHUB_TOKEN
+  # 
+  # Instruction to retrieve your PAT is documented on https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens
+  - policy: ghcr.io/olblak/policies/rancher/docusaurus/kubewarden:latest
```

Comment:
I think it would be better to point to a specific release

Reply:
Pin the version to the latest version `0.0.3` 🙈